int8 num_markers
---
uint8 first_player
